src/services/ws.ts

```diff
@@ -43,12 +62,12 @@ function init(httpServer, sessionParser) {
     });
 
     webSocketServer.on('connection', (ws, req) => {
-        ws.id = utils.randomString(10);
+        (ws as any).id = utils.randomString(10);
 
         console.log(`websocket client connected`);
 
         ws.on('message', async messageJson => {
-            const message = JSON.parse(messageJson);
+            const message = JSON.parse(messageJson as any);
```

Comment:
Isn't this implicitly `any` already?

Reply:
maybe not, I just realized I still haven't successfully npm installed this locally, sqlite doesn't have pre-built binaries for my version of linux and I haven't fixed it yet :rofl: